src/CIQS-CLI-Extension/azext_ciqs/__init__.py

```diff
@@ -18,23 +20,34 @@ def __init__(self, cli_ctx=None):
         super(CiqsCommandsLoader, self).__init__(cli_ctx=cli_ctx, custom_command_type=custom_type)
 
     def load_command_table(self, args):
-        #with self.command_group('ciqs') as g:
 
         with self.command_group('ciqs deployment') as g:
-            g.custom_command('list', 'listDeployments')
+            g.custom_command('list', 'listDeployments', table_transformer=format.transform_deploymentList)
             g.custom_command('create', 'createDeployment')
             g.custom_command('deploy', 'deployDeployment')
             g.custom_command('view', 'viewDeployment')
             g.custom_command('delete', 'deleteDeployment')
 
         with self.command_group('ciqs template') as g:
-            g.custom_command('list', 'listTemplates')
-            g.custom_command('view', 'getTemplate')
+            g.custom_command('list', 'listTemplates', table_transformer=format.transform_templateList)
+            g.custom_command('view', 'getTemplate', table_transformer=format.transform_templateView)
             g.custom_command('locations', 'listLocations')
 
         return self.command_table
 
-    def load_arguments(self, _):
-        pass
+    def load_arguments(self, command):
+        
+        with self.argument_context('ciqs') as c:
+            c.argument('templateId', options_list=('--template-id', '-t'), help='ID of template to use')
```

Comment:
Suggest: 'ID of template to use' -> 'Unique ID of a solution template'

Reply:
done